### PR TITLE
add definition of power efficient, remove battery note

### LINKED ISSUE #44: powerEfficient needs some objective definition
The powerEfficient attribute currently is open ended, so it would be difficult to have multiple UAs respond similarly.  I'm not sure how it can be objectively defined, but that seems needed.

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #44

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/media-capabilities/pull/221.html" title="Last updated on Jul 4, 2024, 9:06 AM UTC (c049038)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/media-capabilities/221/1774c15...c049038.html" title="Last updated on Jul 4, 2024, 9:06 AM UTC (c049038)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -782,13 +782,12 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
...
-      whether the media they would like to play is appropriate for the user
-      configuration. It is worth noting that even when a device is not power
-      constrained, high power usage has side effects such as increasing the
-      temperature or the fans noise.
+    <p>
+      Encoding or decoding is <dfn>power efficient</dfn> when the power draw is optimal or close to optimal. 
+      The definition of optimal power draw from encoding or decoding is left to the user agent but common 
+      implementation strategies would be to consider using hardware as optimal power draw. 
+      It is NOT RECOMMENDED that user agents mark as power efficient hardware encoding or decoding as 
+      non-accelerated codec be as efficient as to accelerated ones (for example, low resolution videos).
```
Comment (marcoscaceres) [inline index.bs:790]:
```suggestion
      non-accelerated codec be as efficient as to accelerated ones (for example, low resolution videos).
       The user agent SHOULD NOT
              take into consideration the current power source in order to
              determine the encoding power efficiency unless the device's power
              source has side effects such as enabling different encoding
              modules.
```

File: index.bs [hunk 2]
```diff
@@ -782,13 +782,17 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
...
```
Comment (chrisn) [inline index.bs:790]:
```suggestion
      User agents SHOULD NOT mark as power efficient hardware encoding or decoding, as 
      non-accelerated codecs can be as efficient as accelerated ones (for example, with low resolution video).
```

File: index.bs [hunk 3]
```diff
@@ -782,13 +782,17 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
...
 
-    <p class='note'>
-      Authors can use {{MediaCapabilitiesInfo/powerEfficient}} in concordance
-      with the Battery Status API [[battery-status]] in order to determine
-      whether the media they would like to play is appropriate for the user
-      configuration. It is worth noting that even when a device is not power
-      constrained, high power usage has side effects such as increasing the
-      temperature or the fans noise.
+    <p>
+      Encoding or decoding is <dfn>power efficient</dfn> when the power draw is optimal or close to optimal.
```
Comment (marcoscaceres) [inline index.bs:786]:
```suggestion
      Encoding or decoding is <dfn>power efficient</dfn> when the power draw is optimal. 
```

File: index.bs [hunk 4]
```diff
@@ -782,13 +782,17 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
...
+      Encoding or decoding is <dfn>power efficient</dfn> when the power draw is optimal. 
+      The definition of optimal power draw from encoding or decoding is left to the user agent but common 
+      implementation strategies would be to consider using hardware as optimal power draw. 
+      User agents SHOULD NOT mark as power efficient hardware encoding or decoding, as 
+      non hardware-accelerated codecs can be as efficient as accelerated ones (for example, with low resolution video).
+       The user agent SHOULD NOT
+              take into consideration the current power source in order to
+              determine the encoding power efficiency unless the device's power
+              source has side effects such as enabling different encoding
+              modules.
```
Comment (marcoscaceres) [inline index.bs:795]:
```suggestion
      Encoding or decoding is considered <dfn>power efficient</dfn> when the
      power draw is optimal. The definition of optimal power draw for encoding
      or decoding is left to the user agent. However, a common implementation
      strategy is to consider hardware usage as indicative of optimal power
      draw. User agents SHOULD NOT mark hardware encoding or decoding as power
      efficient by default, as non-hardware-accelerated codecs can be just as
      efficient, particularly with low-resolution video. User agents SHOULD
      NOT take the device's power source into consideration when determining
      encoding power efficiency unless the device's power source has side
      effects such as enabling different encoding modules.

```

**Review by tidoust — APPROVED** [review]

File: index.bs [hunk 5]
```diff
@@ -782,13 +782,18 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
...
+      power draw is optimal. The definition of optimal power draw for encoding
+      or decoding is left to the user agent. However, a common implementation
+      strategy is to consider hardware usage as indicative of optimal power
+      draw. User agents SHOULD NOT mark hardware encoding or decoding as power
+      efficient by default, as non-hardware-accelerated codecs can be just as
+      efficient, particularly with low-resolution video. User agents SHOULD
+      NOT take the device's power source into consideration when determining
+      encoding power efficiency unless the device's power source has side
+      effects such as enabling different encoding modules.
+
```
Comment (marcoscaceres) [inline index.bs:796]:
```suggestion
```

**Comment by aboba** [comment]
This PR raises the question of whether WebCodecs `isConfigSupported()` needs to return info on power efficiency. We had assumed that hardware acceleration support was sufficient, but your arguments against conflating acceleration and power efficiency seem sound.

**Review by aboba — APPROVED** [review]

File: index.bs [hunk 6]
```diff
@@ -782,13 +782,17 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
...
+    <p>
+      Encoding or decoding is considered <dfn>power efficient</dfn> when the
+      power draw is optimal. The definition of optimal power draw for encoding
+      or decoding is left to the user agent. However, a common implementation
+      strategy is to consider hardware usage as indicative of optimal power
+      draw. User agents SHOULD NOT mark hardware encoding or decoding as power
+      efficient by default, as non-hardware-accelerated codecs can be just as
+      efficient, particularly with low-resolution video. User agents SHOULD
+      NOT take the device's power source into consideration when determining
+      encoding power efficiency unless the device's power source has side
```
Comment (chrisn) [inline index.bs:794]:
```suggestion
      power efficiency unless the device's power source has side
```

File: index.bs [hunk 7]
```diff
@@ -782,13 +782,17 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
...
+      effects such as enabling different encoding modules.
```
Comment (chrisn) [inline index.bs:795]:
```suggestion
      effects such as enabling different encoding or decoding modules.
```

**Review by Djuffin — APPROVED** [review]

**Review by aboba — APPROVED** [review]

**Comment by tidoust** [comment]
@chrisn, you didn't integrate your last suggested edits to drop "encoding" and add "or decoding" where the prose was focused on encoding. Is that intended?

**Comment by chrisn** [comment]
Oops, thanks. I'll fix that.

**Comment by chrisn** [comment]
Fixed in https://github.com/w3c/media-capabilities/commit/35e5eaa1fb13f9dc6667aaecf626a48688f31465.

**Pull request merged**